dbtmetabase/metabase.py

```diff
@@ -37,6 +50,15 @@ def __init__(
         self.protocol = "http" if use_http else "https"
         self.verify = verify
         self.session_id = self.get_session_id(user, password)
+        self.collections: Iterable = []
+        self.tables: Iterable = []
+        self.table_map: MutableMapping = {}
+        self.models_exposed: List = []
+        self.mb_native_query: str = ""
```

Comment:
Don't use the `mb_` prefix. Externally, I moved to `metabase_`, but here you're already in the Metabase context, so no reason for the prefix altogether.

Reply:
removed prefix, thanks!